src/components/app/list/TriggerUrl.tsx

```diff
@@ -156,7 +156,12 @@ export function TriggerUrlModal({ appId, envId, installedAppId, isEAMode, close
     )
 }
 
-export function CopyToClipboardText({ text, iconClass }: { text: string; iconClass?: string }) {
+export function CopyToClipboardText({
```

Comment:
1. Can we extract this component out in helpers as we can reuse it
2. Rename it to more specific like  `CopyToClipboardTextWithTippy`